test/e2e/operator_test.go

```diff
@@ -450,6 +450,12 @@ func TestContourSpec(t *testing.T) {
 	}
 	t.Logf("deleted contour %s/%s", operatorNs, testName)
 
+	// Ensure the envoy service is cleaned up automatically.
```

Comment:
@sunjayBhatia why are you waiting for the envoy svc deletion here?

Reply:
I thought it would help with the timeout waiting for the namespace to be deleted, but its pretty much a no-op since the service deletion has nothing to do with the pod deletion